### DISCUSSION TITLE
Set PowerStream Other Loads

### DISCUSSION THREAD

**Discussion by danieleproietti** [tolwi/hassio-ecoflow-cloud#246] [General]
Hi,
can i set PowerStream Other Loads dinamically from automation? I wish it was the same as my electric meter. Thanks!

**Comment by Nid01** [comment]
Hi @danieleproietti,
since the API behind the integration can only read PowerStream data, you have to try the solution from @svenerbe https://github.com/tolwi/hassio-ecoflow-cloud/issues/124#issuecomment-2033707863 until @Mark-Hicks or I am able to provide an integration using the new API.

**Reply by Mark-Hicks** [reply]
I do not have a Power Stream and never will since it is not available or usable in USA so I do not have plans to work on that for my integration. I am open to sharing code at some point of course but I've had to put this project on the back burner for a while.
